src/components/SearchResults.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import SanitizedHtml from '../containers/SanitizedHtml';
+
+/** */
+export class SearchResults extends Component {
+  /** */
+  render() {
+    const {
+      cwId,
+      searchHits,
+    } = this.props;
+
+    return (
+      <>
```

Comment:
I believe this Fragment is redundant since we have a top-level `<List>`

Reply:
Good catch, I was thinking that there would be additional components coming in for future tickets and hoping that this might be a placeholder. Do you think it's worth keeping in for that?

Reply:
Yeah, that's fine.  No harm, no foul. 😸